manim/utils/bezier.py

```diff
@@ -369,3 +372,114 @@ def get_quadratic_approximation_of_cubic(a0, h0, h1, a1):
 
 def is_closed(points: typing.Tuple[np.ndarray, np.ndarray]) -> bool:
     return np.allclose(points[0], points[-1])
+
+
+def bez_params_from_point(
```

Comment:
I find the shortening of `bezier` to `bez` confusing and without any added value
```suggestion
def bezier_params_from_point(
```

Reply:
Done.